### ISSUE TITLE
User should be able to get positions of devices from the REST client.

### ISSUE THREAD

**Issue by DiamondJoseph** [DiamondLightSource/blueapi#379]
We expose the `move` stub, which takes a mapping of device to value to move them to. But we don't expose what type `value` should be per device, and for those devices with a JSONable value (which is probably only those that are Movable) we currently do not have any way of getting the location of those devices without running a `count` plan.

There should be a REST endpoint that can be called to:
If there is a running task, return 503: Server Busy
Otherwise:
Create a task to read either a selection or all of the Movable devices
Return those values as a JSON blob {"device_name": device_value}.

**Comment by callumforrester** [comment]
Yes, I've been wondering exactly what form this should take, and whether it should work while a plan is running...
On the one hand, making plans as atomic as possible is a good thing, on the other hand asynchronous monitoring would give feature parity with GDA

**Comment by callumforrester** [comment]
Devices are also in the subprocess, see https://github.com/DiamondLightSource/blueapi/issues/371

**Comment by stan-dot** [comment]
what if the ophyd devices stream kafka events?

**Comment by callumforrester** [comment]
Duplicate of #576
